packages/jaeger-ui-components/src/Theme.tsx

```diff
@@ -58,9 +104,14 @@ export function ThemeConsumer(props: ThemeConsumerProps) {
 }
 
 const memoizedThemeMerge = memoizeOne(value => {
+  const darkOverrides: Partial<Theme> = {};
+  if (!isLight(value)) {
+    darkOverrides.servicesColorPalette = COLORS_HEX_DARK;
```

Comment:
what type is value here?  to simplify logic like in the Grafana them I added isLight bool property instead of the ThemeType so no need for functions like isLight

Reply:
It's ThemeOptions or undefined, need to update the types here, interesting implicit any did not kick in here.

> to simplify logic like this in the Grafana them I added isLight bool property instead of the ThemeType so no need for functions like isLight

I can also add it here just seemed to me that it is better not to duplicate information, considering I need to be able to override the theme values from outside and so merge the ThemeOptions with default to get the theme, it would be a bit more typing and checking to make sure I don't get `{ type: "Light", isLight: false}` somewhere and `isLight(theme)` vs `theme.isLight` did not seem that much different.